zenoh/src/api/queryable.rs

```diff
@@ -132,7 +132,7 @@ impl Query {
         self.attachment.as_mut()
     }
 
-    /// Sends a reply in the form of [`Sample`] to this Query.
+    /// Send a reply in the form of [`Sample`] to this Query.
```

Comment:
```suggestion
    /// Sends a reply in the form of [`Sample`] to this Query.
```

I would stick with the third person pronoun as is done in `std` and most Rust crates.

Here's an example: https://doc.rust-lang.org/stable/std/vec/struct.Vec.html#method.push